src/Test.Utilities.Sarif/TestData.cs

```diff
@@ -39,19 +39,20 @@ public static class RuleIds
             public const string Rule10 = "TST0010";
         }
 
-        // TODO: In general, static instances of SARIF logs are a bad idea, because
-        //       they are mutable. A test may munge these and leave them in a bad
-        //       state for another test. Instead, we should prefer factories that
-        //       reliably generate a fresh copy of a log for each test.
-        public static class SarifLogs
+        // In general, static instances of SARIF logs are a bad idea, because
+        // they are mutable. A test may munge these and leave them in a bad
+        // state for another test. Instead, we should prefer factories that
+        // reliably generate a fresh copy of a log for each test.
+
+        public static SarifLog CreateOneIdThreeLocations()
         {
-            public readonly static SarifLog OneIdThreeLocations = new SarifLog
+            return new SarifLog
             {
                 Runs = new[]
-                {
+            {
                     new Run
                     {
-                        Tool = SimpleLog.Runs[0].Tool,
+                        Tool = CreateSimpleLog().Runs[0].Tool,
```

Comment:
>CreateSimpleLog [](start = 31, length = 15)

Instead of creating a whole log for the sake of getting its Tool, consider `CreateSimpleTool`.

Reply:
Good idea, later.

---
In reply to: [395165690](https://github.com/microsoft/sarif-sdk/pull/1818#discussion_r395165690) [](ancestors = 395165690)